style.css

```diff
@@ -8,6 +8,7 @@
 	Stable tag: 2.3.1
 	Requires at least: 4.7
 	Tested up to: 5.6
+	Requires PHP: 5.6
```

Comment:
Do we need that auto-generated string here?
@matipojo

Reply:
No, remove it.

Reply:
Done.